src/stores/RoomListStore.tsx

```diff
@@ -0,0 +1,77 @@
+import { makeAutoObservable, makeObservable } from "mobx";
+import { RootStore } from "@/stores/RootStore";
+import RoomService from "@/service/room.service";
+import { DEFAULT_PASSWORD, DEFAULT_THUMBNAIL } from "@/constants/default";
+import roomService from "@/service/room.service";
+import { RoomOverview } from "@/models/room/RoomOverview";
+
+export class Room {
+  readonly _id: string = "";
+  readonly _master_id: string = "test";
+  readonly _title: string = "test";
+  readonly _thumbnail?: string = "test";
+  readonly _password?: string = "test";
+  readonly _timer: number = 40;
+  readonly _short_break: number = 15;
+  readonly _long_break: number = 20;
+  readonly _long_break_interval: number = 3;
+  readonly _expired_at: string = "2021-08-21T12:30:00.000Z";
+  readonly _room_tag: string[] = [];
+  constructor() {
+    makeAutoObservable(this);
+  }
+}
+
+export class RoomListStore {
+  //TODO 방이 만들어지고 나서 tempRoom을 초기화 해야함
+  private _pageNum: number = 0;
+  private readonly _roomService: RoomService = new RoomService();
+  readonly rootStore: RootStore;
+
+  private rooms: Room[] = [];
+  private tempRoom: Room = new Room();
+  private roomOverviews: RoomOverview[] = [];
+  constructor(root: RootStore) {
+    makeAutoObservable(this);
+    this.rootStore = root;
+  }
+
+  get_created_title() {
```

Comment:
getter는 아래와 같이 만들 수 있습니다. 그리고 함수이름 컨벤션은 통일을 시켜야합니다. 그리고 반환값도 명시하는 게 추후에 코드를 읽을 때 좋습니다.
```suggestion
  get createdTitle(): string {
```